### PR TITLE
Coupe & Sedan keeboards

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description

A 60% board and a 68% board

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clawsome/coupe/info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "keyboard_name": "Coupe",
+    "url": "",
+    "maintainer": "qmk",
```
Comment (tzarc) [inline keyboards/clawsome/coupe/info.json:4]: Please change these to your own URL/name etc.

File: keyboards/clawsome/sedan/info.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "sedan",
+    "url": "",
+    "maintainer": "qmk",
```
Comment (tzarc) [inline keyboards/clawsome/sedan/info.json:4]: Please change these to your own URL/name etc.

File: keyboards/clawsome/coupe/config.h [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (tzarc) [inline keyboards/clawsome/coupe/config.h:24]:
```suggestion
#define PRODUCT_ID      0x7E94
```
```
echo -n "The Coupe" | shasum | cut -c1-4
```

File: keyboards/clawsome/sedan/config.h [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (tzarc) [inline keyboards/clawsome/sedan/config.h:24]:
```suggestion
#define PRODUCT_ID      0x8C78
```
```
echo -n "The Sedan" | shasum | cut -c1-4
```

File: keyboards/clawsome/sedan/sedan.c [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "coupe.h"
```
Comment (tzarc) [inline keyboards/clawsome/sedan/sedan.c:16]: Wrong filename here.

File: keyboards/clawsome/sedan/sedan.h [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+                                                                                      \
```
Comment (zvecr) [inline keyboards/clawsome/sedan/sedan.h:21]:
```suggestion
```

File: keyboards/clawsome/sedan/sedan.h [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D,      K0E,   \
+    K10,   K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D,    K1E,   \
+    K20,    K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D,        K2E,   \
+    K30,       K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,     K3D, K3E,   \
+    K40, K41, K42, K45, K49, K4A, K4B, K4C, K4D, K4E,
```
Comment (zvecr) [inline keyboards/clawsome/sedan/sedan.h:26]:
```suggestion
    K40, K41, K42, K45, K49, K4A, K4B, K4C, K4D, K4E                                  \
```

**Pull request closed without merging**